core/src/main/scala/filodb.core/reprojector/FiloAppendStore.scala

```diff
@@ -51,9 +51,9 @@ class FiloAppendStore(val projection: RichProjection,
   private var wal: Option[WriteAheadLog] = createWalAheadLog
 
   def createWalAheadLog : Option[WriteAheadLog] = {
-    if (!reloadFlag) {
+    if (!reloadFlag && config.getBoolean("write-ahead-log.write-ahead-log-enabled")) {
```

Comment:
I would definitely move this getBoolean thing out to a common private val.... it's used in two places at least